Core/include/Acts/Definitions/Direction.hpp

```diff
@@ -88,16 +92,17 @@ class Direction final {
   ///
   /// @return an opposite direction
   constexpr Direction invert() const {
-    return (m_value == Value::Positive) ? Value::Negative : Value::Positive;
+    return Direction{m_value == Value::Positive ? Value::Negative
+                                                : Value::Positive};
   }
 
   std::string toString() const;
 
   constexpr Direction() = default;
```

Comment:
should we get rid of the default constructor?

Reply:
Might be a good idea, I'll give it a try.